lib/google_places/rectangle.rb

```diff
@@ -0,0 +1,19 @@
+module GooglePlaces
+  class Rectangle
+    def initialize(lat1, lng1, lat2, lng2)
```

Comment:
Isn't it more understandable to use something like `north_west`, `north_east`, `south_east`, and `south_west` here, to indicate the outer points of the rectangle? Latitude and longitude may be a bit too abstract?